Makefile

```diff
@@ -313,7 +313,7 @@ GOFILES:=$(shell go list -f '{{$$p := .}}{{range $$f := .GoFiles}}{{$$p.Dir}}/{{
 .PHONY: gocyclo
 gocyclo:
 	# Run gocyclo over all .go files
-	gocyclo -over 15 ${GOFILES}
+	gocyclo -over 17 ${GOFILES}
```

Comment:
How did you rebase the change? This shouldn't show up as a part of your pr.

Reply:
Your commit was cherry-picked for the feature branch.